src/components/Tipseen/TipseenContent.tsx

```diff
@@ -1,64 +1,77 @@
+import React, { FC } from "react";
 import cx from "classnames";
-import PropTypes from "prop-types";
 import { BEMClass } from "../../helpers/bem-helper";
 import { NOOP } from "../../utils/function-utils";
 import Button from "../../components/Button/Button";
 import { backwardCompatibilityForProperties } from "../../helpers/backwardCompatibilityForProperties";
-import { DISMISS_BUTTON_TEXT, SUBMIT_BUTTON_TEXT } from "./TipseenConstants";
+import { ButtonPropsBackwardCompatible, DISMISS_BUTTON_TEXT, SUBMIT_BUTTON_TEXT } from "./TipseenConstants";
 import TipseenBasicContent from "./TipseenBasicContent";
+import { VibeComponentProps } from "../../types";
+import { ElementContent } from "../../types/ElementContent";
 import styles from "./TipseenContent.module.scss";
 
 const BASE_CSS_CLASS = "monday-style-tipseen-content";
 const bemHelper = BEMClass(BASE_CSS_CLASS);
-const EMPTY_OBJECT = {};
+const EMPTY_OBJECT: ButtonPropsBackwardCompatible = {};
 
-const TipseenContent = ({
+interface TipseenContentProps extends VibeComponentProps {
+  title?: string;
+  isDismissHidden?: boolean;
+  children?: ElementContent;
+  isSubmitHidden?: boolean;
```

Comment:
if we already change this component let's add prop: "hideSubmit" and make this one as backward compatibility prop (same reason)